Libraries/NativeAnimation/RCTAnimatedValueChangeEvent.m

```diff
@@ -0,0 +1,55 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+#import "RCTAnimatedValueChangeEvent.h"
+
+@implementation RCTAnimatedValueChangeEvent
+{
+  CGFloat _value;
+}
+
+@synthesize viewTag = _viewTag;
```

Comment:
Why do not declare this as regular `@`-properties?

Reply:
Same as the _bridge for native modules, it implements properties from the RCTEvent protocol. Based this on other events (https://github.com/facebook/react-native/blob/master/React/Views/RCTScrollView.m#L51)